collector/infiniband_linux.go

```diff
@@ -85,6 +85,38 @@ func NewInfiniBandCollector(logger log.Logger) (Collector, error) {
+		"duplicate_request_total":          "The number of received packets. A duplicate request is a request that had been previously executed.",
+		"implied_nak_seq_errors_total":     "The number of time the requested decided an ACK. with a PSN larger than the expected PSN for an RDMA read or response.",
+		"lifespan_millisecond":             "The maximum period in ms which defines the aging of the counter reads. Two consecutive reads within this period might return the same values.",
+		"local_ack_timeout_errors_total":   "The number of times QP's ack timer expired for RC, XRC, DCT QPs at the sender side. The QP retry limit was not exceed, therefore it is still recoverable error.",
+		"np_cnp_sent_total":                "The number of CNP packets sent by the Notification Point when it noticed congestion experienced in the RoCEv2 IP header (ECN bits). The counters was added in MLNX_OFED 4.1",
+		"np_ecn_marked_roce_packets_total": "The number of RoCEv2 packets received by the notification point which were marked for experiencing the congestion (ECN bits where '11' on the ingress RoCE traffic) . The counters was added in MLNX_OFED 4.1",
+		"out_of_buffer_total":              "The number of drops occurred due to lack of WQE for the associated QPs.",
+		"out_of_sequence_total":            "The number of out of sequence packets received.",
+		"packet_seq_errors_total":          "The number of received NAK sequence error packets. The QP retry limit was not exceeded.",
+		"req_cqe_errors_total":             "The number of times requester detected CQEs completed with errors. The counters was added in MLNX_OFED 4.1",
+		"req_cqe_flush_errors_total":       "The number of times requester detected CQEs completed with flushed errors. The counters was added in MLNX_OFED 4.1",
+		"req_remote_access_errors_total":   "The number of times requester detected remote access errors. The counters was added in MLNX_OFED 4.1",
+		"req_remote_invalid_request_total": "The number of times requester detected remote invalid request errors. The counters was added in MLNX_OFED 4.1",
+		"resp_cqe_errors_total":            "The number of times responder detected CQEs completed with errors. The counters was added in MLNX_OFED 4.1",
+		"resp_cqe_flush_errors_total":      "The number of times responder detected CQEs completed with flushed errors. The counters was added in MLNX_OFED 4.1",
+		"resp_local_length_errors_total":   "The number of times responder detected local length errors. The counters was added in MLNX_OFED 4.1",
+		"resp_remote_access_errors_total":  "The number of times responder detected remote access errors. The counters was added in MLNX_OFED 4.1",
+		"rnr_nak_retry_errors_total":       "The number of received RNR NAK packets. The QP retry limit was not exceeded.",
+		"roce_adp_retrans_total":           "The number of adaptive retransmissions for RoCE traffic. The counter was added in MLNX_OFED rev 5.0-1.0.0.0 and kernel v5.6.0",
```

Comment:
Spell out retransmits

Reply:
Thanks you for your review. Done. Please re-check